pages/people_page.py

```diff
@@ -10,6 +10,8 @@ class PeoplePage(Page):
 
     url = '/people'
     admins = ['Tim Bortnik']
+    list_name = []
+    rletter = ''
```

Comment:
Please name your methods fully, rletter - poor description of the method.

Reply:
replaced